### PR TITLE
Added GitHub Actions

### PR DESCRIPTION
**Pull request by daminijain23**
Added .github/workflows/tox-test.yml and .github/workflows/release.yml.

Updated tox.ini for static -

constrained pylint and astroid version for travis.

### CODE REVIEW

**Review by rbikar — CHANGES_REQUESTED** [review]

File: .github/workflows/tox-test.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+name: Tox tests
+
+on: [push, pull_request]
+
+jobs:
+  py26:
```
[the remark this answers is not in the document]
Comment (daminijain23) [reply .github/workflows/tox-test.yml]: @rbikar 
job for py26 is dropped.
Jobs for py 38 and py39 are added.

File: requirements-test-py26.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+mock==2.0.0
```
Comment (rbikar) [inline requirements-test-py26.txt:1]: @daminijain23  I think this file is not needed anymore.

**Review by rbikar — CHANGES_REQUESTED** [review]

File: requirements-test-py26.txt (hunk 2 above)
Comment (daminijain23) [reply requirements-test-py26.txt]: @rbikar removed

File: test-requirements.txt [hunk 3]
```diff
@@ -1,3 +1,7 @@
 pytest
 mock
-ordereddict; python_version<'2.7'
\ No newline at end of file
+ordereddict; python_version<'2.7'
```
Comment (rajulkumar) [inline test-requirements.txt:3]: May remove conditional requirements if py26 is not supported anymore.

File: tox.ini [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py35,pidiff,static,docs
+envlist = py26,py27,py35,pidiff,static,docs,py38,py39
```
Comment (rajulkumar) [inline tox.ini:2]: May remove py26 here.

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```
Comment (rajulkumar) [inline .github/workflows/release.yml:24]: Should this be secrets.PYPI_TOKEN?

**Comment by daminijain23** [comment]
Included the suggested changes and squashed commits

**Review by rajulkumar — DISMISSED** [review]

**Review by rbikar — APPROVED** [review]
@daminijain23  
Not sure about pinning astroid and pytest version, giving +1 anyway. Let's see if it works properly.

File: ubiconfig/utils/config_validation.py [hunk 6]
```diff
@@ -12,6 +12,6 @@ def validate_config(data, schema=None):
 
     If no schema is provided, :data:`DEFAULT_SCHEMA` is used"""
     if schema is None:
-        with open(DEFAULT_SCHEMA) as f:
-            schema = json.load(f)
+        with open(DEFAULT_SCHEMA) as f:  # pylint: disable=unspecified-encoding
+            schema = json.load(f)  # pylint: disable=unspecified-encoding
```
Comment (rajulkumar) [inline ubiconfig/utils/config_validation.py:16]: "# pylint: disable=unspecified-encoding" is not required here. The error occurs only for open() function and should be specified only in that line.

**Review by rajulkumar — CHANGES_REQUESTED** [review]
pylint's unspecified-encoding should also be disabled for ubiconfig/_impl/loaders/local.py
`ubiconfig/_impl/loaders/local.py:62:13: W1514: Using open without explicitly specifying an encoding (unspecified-encoding)`

**Comment by daminijain23** [comment]
@rbikar @rajulkumar Please review the changes.

**Review by rbikar — APPROVED** [review]
lgtm

**Review by rajulkumar — APPROVED** [review]

**Pull request merged**